cns/restserver/util.go

```diff
@@ -387,13 +383,13 @@ func (service *HTTPRestService) getNetworkContainerResponse(
 			}
 
 			vfpUpdateComplete := !waitingForUpdate
-			ncstatus, _ := service.state.ContainerStatus[containerID]
+			ncstatus := service.state.ContainerStatus[containerID]
```

Comment:
assume map always has ipconfig Or it is ok to use zero value of the return type.